test/TestSimpleCounter.js

```diff
@@ -0,0 +1,15 @@
+const SimpleCounter = artifacts.require("./SimpleCounter.sol");
```

Comment:
We should set up a more robust testing boilerplate, but this is fine for now since we only have boilerplate contracts